src/MICore/Debugger.cs

```diff
@@ -716,7 +748,36 @@ private Task<Results> CmdUnixBreak(int debugeePid, ResultClass expectedResultCla
             return Task.FromResult<Results>(new Results(ResultClass.done));
         }
 
-        #region ITransportCallback implementation
+        private Task<Results> CmdBreakWindows(int debugeePid, ResultClass expectedResultClass)
+        {
+            lock (this._debuggeePids)
+            {
+                foreach (int pid in this._debuggeePids.Values)
+                {
+                    try
+                    {
+                        // Mingw has no kill utility. Instead, use kernel32!DebugBreakProcess. Note that this
+                        // can deadlock if the loader lock is taken by a suspended thread. This approach does
+                        // not work in cygwin
+                        Process p = Process.GetProcessById(pid);
+
+#if CORECLR
+                        WindowsNativeMethods.DebugBreakProcess(p.SafeHandle);
```

Comment:
SafeHandle returns a `Microsoft.Win32.SafeHandles.SafeProcessHandle`, not an `IntPtr`, so I'm pretty sure this wont' work as expected.